### PR TITLE
Add Weather Info title

### PR DESCRIPTION
**Pull request by venkykatta**
New added items
- index.css
- Weather Info title

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -1,11 +1,21 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather</title>
-</head>
-<body>
-    <h1>Hello World</h1>
-</body>
```
Comment (su66uu) [inline templates/index.html:10]: Can I why did you remove this code snippet?

File: templates/index.html [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Hello Bulma!</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">
+    <link rel="stylesheet" href="{{ url_for('static',filename='css/index.css') }}">
```
Comment (su66uu) [inline templates/index.html:8]: This is Awesome

**Review by su66uu — CHANGES_REQUESTED** [review]

File: templates/index.html (hunk 1 above)
Comment (venkykatta) [reply templates/index.html:10]: i want try out bulma template

**Review by su66uu — APPROVED** [review]
